src/credentials/CertificationDeclaration.h

```diff
@@ -82,23 +89,22 @@ struct CertificationElementsWithoutPIDs
     uint16_t dacOriginVendorId                   = VendorId::NotSpecified;
     uint16_t dacOriginProductId                  = 0;
     bool dacOriginVIDandPIDPresent               = false;
+    bool authorizedPAAListPresent                = false;
     char certificateId[kCertificateIdLength + 1] = { 0 };
 };
 
 class CertificationElementsDecoder
 {
 public:
     bool IsProductIdIn(const ByteSpan & encodedCertElements, uint16_t productId);
+    bool HasAuthorizedPAA(const ByteSpan & encodedCertElements, const ByteSpan & authorizedPAA);
 
 private:
-    CHIP_ERROR PrepareToReadProductIdList(const ByteSpan & encodedCertElements);
+    CHIP_ERROR FindAndEnterArray(const ByteSpan & encodedCertElements, TLV::Tag listTag);
```

Comment:
Needs to be documented.  In particular, where it positions mReader on success/failure...

Reply:
done